src/regtech_mail_api/settings.py

```diff
@@ -23,6 +23,9 @@ class EmailApiSettings(BaseSettings):
     cc: set[EmailStr] | None = None
     bcc: set[EmailStr] | None = None
 
+    prod_body_template: str = ""
+    beta_body_template: str = ""
```

Comment:
I think we want to make these default to None.  Probably preferred to error out if these env vars aren't set than to send a blank email.  The error we can at least see in the logs, whereas the format code will just pass the empty string along.